renderer/src/pages/Dashboard.tsx

```diff
@@ -78,7 +78,7 @@ const Dashboard = (): JSX.Element => {
               <p className="w-fit text-header-m font-bold font-number total-jobs" title="total jobs">{totalJobs}</p>
             </div>
             <div className="mb-6">
-              <p className="w-fit text-body-3xs text-grayscale-700 uppercase">Total Earnings (updated daily)</p>
+              <p className="w-fit text-body-3xs text-grayscale-700 uppercase">Total Earnings (comming soon)</p>
```

Comment:
I think the correct spelling is `coming`?

```suggestion
              <p className="w-fit text-body-3xs text-grayscale-700 uppercase">Total Earnings (coming soon)</p>
```